src/core/urdb.jl

```diff
@@ -313,8 +313,8 @@ Parse monthly ("flat") and TOU demand rates
 """
 function parse_demand_rates(d::Dict, year::Int; bigM=1.0e8, time_steps_per_hour::Int)
```

Comment:
We could remove `bigM` from these function inputs, right?

Reply:
I'm using them now here: https://github.com/NREL/REopt.jl/commit/3d623f007ed7390c8303195f67f362499564c214

I think we will want this to stay because the bigM's are still going to be necessary when we have loads (they'll just be customized bigM's for a given instance rather than a fixed large number).